### PR TITLE
ZH-trans: add cluster.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by fleeto** [comment]
`[trans-bot:pushed]`

Thank you @xichengliudui, the [related task issue](https://github.com/k8smeetup/website-tasks/issues/737) had been updated.

File: content/zh/docs/tasks/administer-federation/cluster.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+<!--
+This guide explains how to use Clusters API resource in a Federation control plane.
+
+Different than other Kubernetes resources, such as Deployments, Services and ConfigMaps,
+clusters only exist in the federation context, i.e. those requests must be submitted to the
+federation api-server.
+-->
+本指南介绍了如何在联邦控制平面中使用集群 API 资源。
+
+与 Deployment、Service 和 ConfigMap 等 Kubernetes 资源不同，集群只存在于联邦上下文中，即这些请求必须提交给联邦 api-server。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-federation/cluster.md]: 这里的 Cluster 不要翻译，是 federation 上下文中的一种资源

File: content/zh/docs/tasks/administer-federation/cluster.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+<!--
+Starting in Kubernetes 1.7, there is alpha support for directing objects across the federated clusters with the annotation `federation.alpha.kubernetes.io/cluster-selector`. The *ClusterSelector* is conceptually similar to `nodeSelector`, but instead of selecting against labels on nodes, it selects against labels on federated clusters.
+
+The annotation value must be JSON formatted and must be parsable into the [ClusterSelector API type](/docs/reference/federation/v1beta1/definitions/#_v1beta1_clusterselector). For example: `[{"key": "load", "operator": "Lt", "values": ["10"]}]`. Content that doesn't parse correctly will throw an error and prevent distribution of the object to any federated clusters. Objects of type ConfigMap, Secret, Daemonset, Service and Ingress are included in the alpha implementation.
+
+Here is an example ClusterSelector annotation, which will only select clusters WITH the label `pci=true` and WITHOUT the label `environment=test`:
+-->
+
+从 Kubernetes 1.7 开始，alpha 支持使用注解在联邦集群中引导对象 `federation.alpha.kubernetes.io/cluster-selector`。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-federation/cluster.md]: 这句重翻

File: content/zh/docs/tasks/administer-federation/cluster.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+*ClusterSelector* 在概念上类似于 `nodeSelector`，但是它不是针对节点上的标签进行选择，而是针对联邦集群上的标签进行选择。
+
+注解值值必须是 JSON 格式并且必须可解析为 [ClusterSelector API 类型](/docs/reference/federation/v1beta1/definitions/#_v1beta1_clusterselector)。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-federation/cluster.md]: 值

File: content/zh/docs/tasks/administer-federation/cluster.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+`--context=federation` 参数告诉 kubectl 将请求提交给联邦 apiserver，
+而不是将其发送给 Kubernetes 集群。如果您将其提交给 k8s 集群，则会收到错误消息
+
+
+```the server doesn't have a resource type "clusters"```
+
+<!--
+If you passed the correct Federation context but received a message error saying
+-->
+如果您传递了正确的联邦上下文，但是收到了一条消息错误
```
Comment (chenrui333) [inline content/zh/docs/tasks/administer-federation/cluster.md]: 不知道为什么 颜色 是蓝色，可以新开一行。

File: content/zh/docs/tasks/administer-federation/cluster.md [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+<!--
+## Labeling Clusters
+-->
+
+## 标记集群
+
+<!--
+You can label clusters the same way as any other Kubernetes object, which can help with grouping clusters and can also be leveraged by the ClusterSelector.
+-->
+您可以使用与其他任何 Kubernetes 对象相同的方法标记集群，这有助于对集群进行分组，也可以由 ClusterSelector 使用。
```
Comment (chenrui333) [inline content/zh/docs/tasks/administer-federation/cluster.md]:
```suggestion
您可以使用与其他任何 Kubernetes 对象相同的方法标记集群，这有助于对集群进行分组，也可以配合使用 ClusterSelector。
```

File: content/zh/docs/tasks/administer-federation/cluster.md [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+
+*key* 与联邦集群上的标签名称匹配。
+
+*values* 与联邦集群上的标签值匹配。
+
+可能的*操作符*有：`In`、`NotIn`、`Exists`、`DoesNotExist`、`Gt`、`Lt`。
+
+*values* 字段在指定 `Exists` 或 `DoesNotExist` 时为空，在使用 `In` 或 `NotIn` 时可能包含多个字符串。
+
+Currently, 目前，只有整数支持 `Gt` 或 `Lt`。
```
Comment (chenrui333) [inline content/zh/docs/tasks/administer-federation/cluster.md]: 😄 
```suggestion
目前，只有整数支持 `Gt` 或 `Lt`。
```

**Comment by tengqm** [comment]
@xichengliudui Please help address the comments. Thanks.

**Comment by xichengliudui** [comment]
has been updated .

**Comment by tengqm** [comment]
Preview is at: https://deploy-preview-13500--k8s-v1-12.netlify.com/zh/docs/tasks/administer-federation/cluster/

Looks good now.

/approve

**Pull request merged**